<launch>
  <node name="aruco_detector" pkg="mogi_chess_vision" type="aruco_detector.py" respawn="false" output="screen">
  </node>

  <node name="split_and_save" pkg="mogi_chess_vision" type="split_and_save.py" respawn="false" output="screen">
  </node>

</launch>